client/state/comments/actions.js

```diff
@@ -20,6 +20,8 @@ import {
 } from '../action-types';
 import { NUMBER_OF_COMMENTS_PER_FETCH } from './constants';
 
+export const requestComment = ( { siteId, commentId } ) => ( { siteId, commentId } );
```

Comment:
yep! kind of thing thats harder to find because its dead-ish code until it gets used. typescript would solve this :)